wrappers/ios/demo/demo.xcodeproj/project.pbxproj

```diff
@@ -27,16 +42,16 @@
 		388BF034283CC49E005CE271 /* Base */ = {isa = PBXFileReference; lastKnownFileType = file.storyboard; name = Base; path = Base.lproj/LaunchScreen.storyboard; sourceTree = "<group>"; };
 		388BF036283CC49E005CE271 /* Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; path = Info.plist; sourceTree = "<group>"; };
 		388BF043283CE0AC005CE271 /* README.md */ = {isa = PBXFileReference; lastKnownFileType = net.daringfireball.markdown; path = README.md; sourceTree = "<group>"; };
+		52A975492ADADA1B002D6BD8 /* zxing-cpp */ = {isa = PBXFileReference; lastKnownFileType = wrapper; name = "zxing-cpp"; path = "/Users/john/git/private/zxing-cpp"; sourceTree = "<absolute>"; };
```

Comment:
This line definitively needs to change, even if the `path` part should be functionally irrelevant.

Reply:
OH DAMN IT, I reviewed every single change sigh :) commit incoming.